software/tests/conftest.py

```diff
@@ -0,0 +1,6 @@
+import sys
+from pathlib import Path
+
+sys.path.append(str(Path(__file__).parent.parent / 'firmware'))
```

Comment:
Can you help me understand this syntax? How is there a `/` in the middle of `str(...)`?

Reply:
The left operand isn't a string, it's a `Path`, and that class overrides the `/` operator to allow this syntax. The resulting `Path` object is then converted into a string.

I had to look it up too when I found the example code that I copied this from.

Reply:
Lol, python being python.